code/game/jobs/job/security.dm

```diff
@@ -132,9 +127,21 @@
 		if(H.mind.role_alt_title && H.mind.role_alt_title == "Forensic Technician")
 			H.equip_to_slot_or_del(new /obj/item/clothing/suit/storage/forensics/blue(H), slot_wear_suit)
 			H.equip_to_slot_or_del(new /obj/item/weapon/storage/briefcase/crimekit, slot_r_hand)
+			if(istajara(H))
+				H.equip_to_slot_or_del(new /obj/item/clothing/gloves/latex/tajara(H), slot_gloves)
+			if(isunathi(H))

```

Comment:
Doesn't look like this was your doing, but this should be an `else if`, not `if`.